jvm/src/main/kotlin/dom.kt

```diff
@@ -79,7 +79,21 @@ class HTMLDOMBuilder(val document : Document) : TagConsumer<Element> {
     override fun finalize() = lastLeaved ?: throw IllegalStateException("No tags were emitted")
 
     override fun onTagContentUnsafe(block: Unsafe.() -> Unit) {
-        throw UnsupportedOperationException("unsafe content for DOM at JVM is not supported yet")
+        UnsafeImpl.block()
+    }
+
+    fun String.toElement() = DocumentBuilderFactory
+            .newInstance()
+            .newDocumentBuilder()
+            .parse(ByteArrayInputStream(this.toByteArray()))
```

Comment:
what about character encoding here? shouldn't we configure document builder or use `StringReader` instead ?